### PR TITLE
change navigation block description

### LINKED ISSUE #25013: Update the Navigation block's description
### Description

Part of #24875
Updated the Navigation block's description to "A collection of blocks that allow visitors to get around your site."

### PR DESCRIPTION
**Pull request by devfle**
## Description

Fixes #25013 - In this PR I changed the description of the navigation block.

## How has this been tested?

I have uploaded the changes on my local Wordpress installation and checked the changes. Everything worked fine.

## Screenshots 
Before:
before

After:
after

## Types of changes
New feature (non-breaking change which adds functionality) 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/navigation/index.js [hunk 1]
```diff
@@ -47,23 +47,23 @@ export const settings = {
...
+					label: __('About'),
 					url: 'https://make.wordpress.org/',
 				},
 			},
 			{
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'Contact' as in a website's contact page.
-					label: __( 'Contact' ),
+					label: __('Contact'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:66]: Spaces should not be removed as per the WordPress Coding Standards
```suggestion
					label: __( 'Contact' ),
```

File: packages/block-library/src/navigation/index.js [hunk 2]
```diff
@@ -47,23 +47,23 @@ export const settings = {
...
+					label: __('Home'),
 					url: 'https://make.wordpress.org/',
 				},
 			},
 			{
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'About' as in a website's about page.
-					label: __( 'About' ),
+					label: __('About'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:58]:
```suggestion
					label: __( 'About' ),
```

File: packages/block-library/src/navigation/index.js [hunk 3]
```diff
@@ -47,23 +47,23 @@ export const settings = {
 				name: 'core/navigation-link',
 				attributes: {
 					// translators: 'Home' as in a website's home page.
-					label: __( 'Home' ),
+					label: __('Home'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:50]:
```suggestion
					label: __( 'Home' ),
```

File: packages/block-library/src/navigation/index.js [hunk 4]
```diff
@@ -17,26 +17,26 @@ const { name } = metadata;
...
+			title: __('Navigation (horizontal)'),
+			description: __('Links shown in a row.'),
 			attributes: { orientation: 'horizontal' },
 		},
 		{
 			name: 'vertical',
-			title: __( 'Navigation (vertical)' ),
-			description: __( 'Links shown in a column.' ),
+			title: __('Navigation (vertical)'),
+			description: __('Links shown in a column.'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:39]:
```suggestion
			title: __( 'Navigation (vertical)' ),
			description: __( 'Links shown in a column.' ),
```

File: packages/block-library/src/navigation/index.js [hunk 5]
```diff
@@ -17,26 +17,26 @@ const { name } = metadata;
...
+	keywords: [__('menu'), __('navigation'), __('links')],
 
 	variations: [
 		{
 			name: 'horizontal',
 			isDefault: true,
-			title: __( 'Navigation (horizontal)' ),
-			description: __( 'Links shown in a row.' ),
+			title: __('Navigation (horizontal)'),
+			description: __('Links shown in a row.'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:33]:
```suggestion
			title: __( 'Navigation (horizontal)' ),
			description: __( 'Links shown in a row.' ),
```

File: packages/block-library/src/navigation/index.js [hunk 6]
```diff
@@ -17,26 +17,26 @@ const { name } = metadata;
...
-	title: __( 'Navigation' ),
+	title: __('Navigation'),
 
 	icon,
 
-	description: __( 'Add a navigation block to your site.' ),
+	description: __('A collection of blocks that allow visitors to get around your site.'),
 
-	keywords: [ __( 'menu' ), __( 'navigation' ), __( 'links' ) ],
+	keywords: [__('menu'), __('navigation'), __('links')],
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:26]:
```suggestion
	keywords: [ __( 'menu' ), __( 'navigation' ), __( 'links' ) ],
```

File: packages/block-library/src/navigation/index.js [hunk 7]
```diff
@@ -17,26 +17,26 @@ const { name } = metadata;
 export { metadata, name };
 
 export const settings = {
-	title: __( 'Navigation' ),
+	title: __('Navigation'),
 
 	icon,
 
-	description: __( 'Add a navigation block to your site.' ),
+	description: __('A collection of blocks that allow visitors to get around your site.'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:24]:
```suggestion
	description: __( 'A collection of blocks that allow visitors to get around your site.' ),
```

File: packages/block-library/src/navigation/index.js [hunk 8]
```diff
@@ -17,26 +17,26 @@ const { name } = metadata;
 export { metadata, name };
 
 export const settings = {
-	title: __( 'Navigation' ),
+	title: __('Navigation'),
```
Comment (aristath) [inline packages/block-library/src/navigation/index.js:20]:
```suggestion
	title: __( 'Navigation' ),
```

**Review by aristath — CHANGES_REQUESTED** [review]
Looks good, the only thing that needs fixed is coding standards :+1:

**Comment by devfle** [comment]
Hmm..! That comes from my VS-Code Prettier. I will fix this. Thank you!

**Review by aristath — APPROVED** [review]
Thank you for the changes :)
LGTM :+1:

**Pull request merged**